turf/src/commonMain/kotlin/io/github/dellisd/spatialk/turf/Booleans.kt

```diff
@@ -0,0 +1,114 @@
+@file:OptIn(ExperimentalTurfApi::class)
+
+package io.github.dellisd.spatialk.turf
+
+import io.github.dellisd.spatialk.geojson.*
+import kotlin.jvm.JvmOverloads
+
+/**
+ * Takes a [Point] and a [Polygon] and determines if the point
+ * resides inside the polygon. The polygon can be convex or concave. The function accounts for holes.
+ *
+ * @param point input point
+ * @param polygon input polygon
+ * @param ignoreBoundary True if polygon boundary should be ignored when determining if
+ * the point is inside the polygon otherwise false.
+ * @return `true` if the Position is inside the Polygon; `false` if the Position is not inside the Polygon
+ */
+@JvmOverloads
+fun booleanPointInPolygon(point: Point, polygon: Polygon, ignoreBoundary: Boolean = false): Boolean {
```

Comment:
```suggestion
fun booleanPointInPolygon(point: Position, polygon: Polygon, ignoreBoundary: Boolean = false): Boolean {
```
The original turf function accepts a `Coord` which is best matched here by `Position`. This avoids needing to create a new `Point` geometry object _just_ to use this function.